src/app/admin/events/projects-add-modal.component.html

```diff
@@ -27,11 +27,13 @@ <h5 class="modal-title">{{ 'T_ACTION_ADD_PROJECTS_TO_EVENT' | translate }}</h5>
         <form class="form form-horizontal"
               (ngSubmit)="submit(); eventProjectsForm.reset()"
               #eventProjectsForm="ngForm">
-          <div class="form-group row">
+          <div class="form-group row"
+               [ngClass]="{'has-success': projects.valid, 'has-danger': !projects.valid && projects.touched }">
```

Comment:
To avoid checking on inverse state of forms like `!projects.valid` you can directly use `projects.invalid`